docs/management/admin/response-actions-config.asciidoc

```diff
@@ -6,8 +6,68 @@
 :frontmatter-tags-content-type: [how-to]
 :frontmatter-tags-user-goals: [manage]
 
+Endpoint response actions involving third-party systems require additional configuration. This page explains the high-level steps you'll need to take to enable these response actions.
+
+[discrete]
+[[configure-sentinelone-response-actions]]
+== Configure SentinelOne response actions
+
+SentinelOne response actions allow you to perform bidirectional actions on protected hosts, such as directing SentinelOne to isolate a suspicious endpoint from your network, without needing to leave the {elastic-sec} UI.
+
+preview::[]
+
+.Prerequisites
 [sidebar]
 --
-[.text-center]
-**This is a placeholder for future documentation.**
+* https://www.elastic.co/pricing[Subscription level]: Enterprise 
+
+* {kibana-ref}/kibana-role-management.html#adding_kibana_privileges[{kib} feature privilege]: **Actions and Connectors : All**.
+
+* <<endpoint-management-req,{elastic-sec} feature privileges>>: **All** for any response action features, such as **Host Isolation**, that you want to perform.
+
+* Endpoints must have actively running SentinelOne agents installed.
 --
+
+Configuration requires the following general steps. Expand the steps and follow the links for detailed instructions:
+
+. **Generate an API access token in SentinelOne.** You'll need this token in later steps, and it allows {elastic-sec} to collect data and perform actions in SentinelOne. You must create the token with the SentinelOne permissions required for the actions you want to perform (for example, isolating and releasing an endpoint).
```

Comment:
A few thoughts on here:

I doubt that a customer would use the same API key for both the Integration setup (which enables data to be collected from SentinelOne to Elasticsearch) **and** for the connectors setup (which enables the ability for a user to send response actions to the sentinelOne agents).  In my opinion, I would not advise a customer to enter an API key that can do response actions when setting up the integration in fleet. 

You might want to expand on this a little more to indicate that. So basially:

- for the S1 integration: API key needs to at least be able to read data out of sentinelone's system
- for the connector: the API key need to be able to execute response actions (in this release, `isoalte` and `release` type of response actions)
-

Reply:
Gotcha, so they'll actually need two separate API keys for SentinelOne, each scoped for least privilege needed for the integration and the connector. I'll push some edits that address this and the other feedback comments; that might cause GitHub to close or hide some of the conversations, we'll see.